httpie/manager/__main__.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import sys
+
+from typing import List, Union
+
+from httpie.context import Environment
+from httpie.status import ExitStatus
+from httpie.manager.cli import parser
+from httpie.manager.core import MSG_COMMAND_CONFUSION, program as main_program
+
+
+def is_http_command(args: List[Union[str, bytes]], env: Environment) -> bool:
+    """Check whether http/https parser can parse the arguments."""
+
+    from httpie.cli.definition import parser as http_parser
+    from httpie.manager.cli import COMMANDS
+
+    # If the user already selected a top-level sub-command, never
+    # show the http/https version. E.g httpie plugins pie.dev/post
+    if (
+        len(args) >= 1
+        and args[0] in COMMANDS
+    ):
```

Comment:
Is this Black’s doing? :D

Reply:
Nop, this is my muscle memory automatically expanding 2 expressions in an if statement's test :D Black would've probably left them in the same line since it is within 88-chars. Fixed now!